### DISCUSSION TITLE
Welcome to ha-bouncie Discussions!

### DISCUSSION THREAD

**Discussion by mandarons** [mandarons/ha-bouncie#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cap60552** [comment]
Hello all!  My name is John Wohlers.  

I've been working with various home automation tech for ~25 years.  I've got two HA instances, one at my house, and one for my elderly mother.  

Having a real bouncie integration has been a long time dream, I've debated trying to create it myself but I'm just not that capable with python.   

We use bouncie / HA automations to trigger lighting when we get home, make sure the doors are locked when we leave, and I hope now that I don't have to rely on IFTTT I can add a few more features such as reminding me to get gas in the morning if the gas tank is low as part of my good morning routine. 

You may also see me post as @29w211barnes, when I forget to switch from my HA service account back to my personal github account.

**Comment by eralbright** [comment]
I'm trying to set up this bouncie tracker and don't know what to put into the redirect_uri field.
Can you assist me with how to locate this information?

**Reply by mandarons** [reply]
From README:

> Provide redirect URL to be your Home Assistant URL
